package.json

```diff
@@ -18,16 +18,15 @@
   "author": "mapzen",
   "license": "MIT",
   "dependencies": {
-    "event-stream": "^3.3.2",
-    "jshint": "^2.8.0",
     "lodash": "^3.10.1",
-    "pelias-model": "pelias/model#2125cbebd44e1e419831a5bb4405c37375618dde",
-    "precommit-hook": "^3.0.0",
+    "pelias-logger": "0.0.8",
     "through2": "^2.0.0"
   },
   "devDependencies": {
```

Comment:
Aren't some of these still dependencies, not dev-dependencies? Pelias model at least is needed when this package is installed somewhere to be used, not just to be worked on, right?